canonical-websites/www.ubuntu.com
# Create general dynamic form

## Done
Created the [general form](https://docs.google.com/document/d/1JRwDbIk9NzOiUuZvWfcmkXtWxr0An6tpvS6Yt1WPVI0/edit#) and added it to all the contact us buttons/links. 
Split the contact-model styles into its own file.
Click when modal back drop is clicked.

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Go sections of the site other then core and iot and click the contact buttons/links
- See that the modal opens
- Step through it and submit

## Issue / Card
Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/4607
